src/js/components/FloatingElement.jsx

```diff
@@ -422,12 +422,16 @@ export class FloatingElement extends Component {
     }
 
     componentRendered() {
-        // As soon as possible, we need to re-calculate our position
-        this.validatePositioningScheduled = true;
-        window.requestAnimationFrame(() => {
-            this.validatePositioningScheduled = false;
+        if (this.props.positionImmediate) {
```

Comment:
This is OK for now but I think we should re-visit it in the future, perhaps have it only turned on based on detected browser, or perhaps set the popup's style.opacity to 0 initially, then set it to 1 once the position has been set.

Actually, setting the initial values for position to something off-screen may make the problem go away entirely, I haven't yet got a Win10 VM set up to test it.